package.json

```diff
@@ -51,5 +51,8 @@
   },
   "beachball": {
     "gitTags": false
+  },
+  "engines": {
+    "node": ">12.0.0"
```

Comment:
Both Windows Server 2016 and 2019 images are using Node 12, so this should be safe.